Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -321,16 +321,23 @@ extension Generator {
         entity.properties = try makeInlineProperties(for: name, object: details, context: context)
             .filter { !$0.type.isVoid }
             .removingDuplicates(by: \.name) // Sometimes Swifty bool names create dups
-        entity.protocols = getProtocols(for: name, context: context)
+        entity.protocols = getProtocols(for: entity, context: context)
+        
         return entity
     }
     
-    private func getProtocols(for type: TypeName, context: Context) -> Protocols {
+    private func getProtocols(for entity: EntityDeclaration, context: Context) -> Protocols {
```

Comment:
I just went ahead and will pass in the entire entity, mainly because I _think_ we will have to check the nested types for `Hashable` declaration. I will take a look at that next and if not, I will change to just passing the properties there.